packages/components/src/interfaces/internal/CollectionCard.ts

```diff
@@ -3,6 +3,8 @@ import type { IsomerSiteProps, LinkComponentType } from "~/types"
 
 export interface Tag {
   label: string
```

Comment:
what does label represent here? is it the current value that the tag holds?

Reply:
Would it make more sense to call values and currentValues than "label"?

Reply:
> yea, i thought this was a distinct feature of multi-tags to make it useful for things like TOSP. Currently, we are duplicating entries based on the body parts.  

ah ok, can! i'll do it in this pr - thought having singly tagged values w support for multiple topics was enough for the cls use case.

Reply:
updated the filtering behaviour

Reply:
i'll update `label` -> `selected`! i think the `curValues` might be abit misleading, especialy in conjunction with our `setState`